### PR TITLE
Change Spell url to snapshot-8

### PR DESCRIPTION
**Pull request by sravan-s**
### What is this PR for?
The link to Spell documentation go to an undefined page 

### What type of PR is it?
[Bug Fix]

### Todos
* [x] - Add proper link to spell package

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-3041

### How should this be tested?
* Build gh-pages - Go to -> Helium in top menu -> Click "What is Apache Zeppelin Spell"

### Questions:
* Does the licenses files need update? N
* Is there breaking changes for older versions? N
* Does this needs documentation? N

### CODE REVIEW

File: helium_packages.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ limitations under the License.
 From Zeppelin-0.7, you can load/unload a pluggable Apache Zeppelin package on runtime through [Helium framework](https://issues.apache.org/jira/browse/ZEPPELIN-533) in Zeppelin.
 Here are the lists of Helium packages registered in the [NPM Registry](https://docs.npmjs.com/misc/registry) and [Maven Central Repository](http://central.sonatype.org/).
 If you need more information about how you can use the below packages in Zeppelin, see [What is Apache Zeppelin Visualization](https://zeppelin.apache.org/docs/latest/development/writingzeppelinvisualization.html)
-and [What is Apache Zeppelin Spell](https://zeppelin.apache.org/docs/snapshot/development/writingzeppelinspell.html).
+and [What is Apache Zeppelin Spell](https://zeppelin.apache.org/docs/0.8.0-SNAPSHOT/development/helium/writing_spell.html).
```
Comment (AhyoungRyu) [inline helium_packages.md]: Just `snapshot` will work

Comment (sravan-s) [reply helium_packages.md]: @AhyoungRyu changed

**Comment by AhyoungRyu** [comment]
LGTM

**Comment by AhyoungRyu** [comment]
Merge to gh-pages branch if there is no more discussions

**Comment by jongyoul** [comment]
Can you guys share the status of this issue?

**Comment by jongyoul** [comment]
This seemed to be fixed. I'll close this PR if there's no update

**Comment by sravan-s** [comment]
Isn't the issue still remains?

**Comment by jongyoul** [comment]
In 0.8, I’ve found it’s fixed. Can you check it again?

**Comment by jongyoul** [comment]
If no objection will close it.

**Comment by sravan-s** [comment]
sure

**Comment by jongyoul** [comment]
BTW, please close this PR. :-)

**Pull request closed without merging**